.github/workflows/pr.yaml

```diff
@@ -99,15 +92,6 @@ jobs:
             diff -r ../integration-service /tmp/integration-service
             exit 1
           fi
-      - name: Check kcp manifests
-        run: |
-          CONTROL_PLANE=kcp make generate manifests
-          if [[ ! -z $(git status ':(exclude)bin/kustomize' ':(exclude)bin/controller-gen' -s) ]]
-          then
-            echo "generated kcp sources are not up to date:"
-            git --no-pager diff ':(exclude)bin/kustomize' ':(exclude)bin/controller-gen'
-            exit 1
-          fi
       - name: Check kubernetes manifests
         run: |
           CONTROL_PLANE=kubernetes make generate manifests
```

Comment:
I think you can remove the `CONTROL_PLANE` variable in the Makefile

Reply:
Thanks! Updated.